changelogs/fragments/483-ec2_vol_fix_returned_changed_var.yml

```
@@ -0,0 +1,6 @@
+bugfixes:
+- >-
+  ec2_vol - Fixes when ``modify_volume`` is used, but no new disk is being attached, 
+  the module incorrectly reports that no change has occurred even when disks 
+  have been modified (iops, throughput, type, etc.).
+  (https://github.com/ansible-collections/amazon.aws/pull/483).
```

```suggestion
  (https://github.com/ansible-collections/amazon.aws/issues/482).
```

For bugfixes where we have an issue, it's good to link to the issue as it often has far more information about the exact problem (for example stack traces)